apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -96,7 +96,7 @@ const buttonStoryMeta: ButtonStoryMeta = {
 
 export default buttonStoryMeta;
 
-const Template : TemplateFunction<ButtonProps> = ({
```

Comment:
did you manually change this? 
asking as mu vs code auto format might not be working if not 👀

Reply:
It changes on autosave for me, so assume it's an eslint thing. Could just be my VSCode settings though equally, it likes to mess with me sometimes 🤷 😆